### PR TITLE
epic-games-launcher: Add version 14.2.1

### LINKED ISSUE #610: (Request) add Epicgameslauncher
PackageIdentifier: epicgameslauncher
PackageVersion: 1.2.17
PackageLocale: en-US
Publisher: Epic Games Store
PackageName: epicgameslauncher
License: proprietary
ShortDescription: official epic game client
Installers:

Architecture: x64
InstallerType: msi
InstallerUrl: https://launcher-public-service-prod06.ol.epicgames.com/launcher/api/installer/download/EpicGamesLauncherInstaller.msi
ManifestType: singleton
ManifestVersion: 1.0.0

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #610

- [x] I have read the [Contributing Guide](https://github.com/Calinou/scoop-games/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/epic-games-launcher-np.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "14.2.1",
+    "homepage": "https://store.epicgames.com/",
+    "description": "The official launcher for the Epic Games Store.",
```
Comment (Calinou) [inline bucket/epic-games-launcher-np.json:4]:
```suggestion
    "description": "The official launcher for Epic Games Store",

```

**Comment by Calinou** [comment]
Is the `-np` suffix needed? Most other non-portable apps in this bucket don't have it.

**Comment by Zliced13** [comment]
I thought it needed the suffix because the app installs telemetry services that are beyond Scoop's control. But if the suffix is not needed, do you mind removing it please?

**Comment by Zliced13** [comment]
Never mind, don't worry about it, I did it just now.

File: bucket/epic-games-launcher.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "14.2.1",
+    "homepage": "https://store.epicgames.com/",
+    "description": "The official launcher for Epic Games Store",
+
```
Comment (Calinou) [inline bucket/epic-games-launcher.json:5]:
```suggestion
    "description": "The official launcher for Epic Games Store",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**